### DISCUSSION TITLE
Importing into Unreal Engine

### DISCUSSION THREAD

**Discussion by OfficialCRUGG** [BramStoutProductions/MiEx#5] [Q&A]
Did anyone have any luck importing a map exported through MiEx into Unreal Engine 5?
I have tried it importing it through the "USD Stage", but there I get no textures, only solid colors.
When trying to import all the USD files I got exported directly, I get countless errors, and the chunks one even fail.

**Comment by 9kdr** [comment]
Did you find a fix?

**Reply by 9kdr** [reply]
Oh nevermind, if you're still struggling, just use the importer and also use the UnrealEngine texture pack.
